src/get-started/create-service.md

```diff
@@ -371,72 +362,109 @@ executed:
 Let’s define the codes of the above errors:
 
 ```rust
-#[derive(Debug, Fail)]
-#[repr(u8)]
+/// Error codes emitted by `TxCreateWallet` and/or `TxTransfer`
+/// transactions during execution.
+#[derive(Debug, ExecutionFail)]
 pub enum Error {
-    #[fail(display = "Wallet already exists")]
+    /// Wallet already exists.
+    ///
+    /// Can be emitted by `TxCreateWallet`.
```

Comment:
Nit: IMO, "Can be emitted..." description can be removed in order to shorten the code sample.